packages/auth/index.rn.ts

```diff
@@ -51,16 +51,37 @@ export { PhoneMultiFactorGenerator } from './src/platform_browser/mfa/assertions
 
 export { getReactNativePersistence };
 
+const NO_PERSISTENCE_WARNING = `
+You are initializing Firebase Auth for React Native without providing
+AsyncStorage. Auth state will default to memory persistence and will not
+persist between sessions. In order to persist auth state, install the package
+"@react-native-async-storage/async-storage" and provide it to
+initializeAuth:
+
+import { initializeAuth, getReactNativePersistence } from 'firebase/auth';
+import ReactNativeAsyncStorage from '@react-native-async-storage/async-storage';
+const auth = initializeAuth(app, {
```

Comment:
Is this still required?

Reply:
I think we're going from it not being required for the user to do explicitly (v10) to it being required (this PR reverting some of the v10 changes). We're not doing it anymore, and we're asking the user to do it.